src/util.ts

```diff
@@ -139,7 +139,7 @@ async function addManifestToSideloadingDirectory(application: string, manifestPa
     return fs.ensureLinkSync(manifestPath, sideloadingManifestPath);
 
   } catch (err) {
-    throw new Error(`addManifestToSideloadingDirectory failed with error:\n${err}`);
+    throw new Error(`Unable to add manifest to the sideloading directory\n${err}`);
```

Comment:
nit: Can you please add "the" and a period":  Unable to add **the** manifest to the sideloading directory.

The other message already has it.